### ISSUE TITLE
Dataloader Shuffling

### ISSUE THREAD

**Issue by OAHC2022** [joaoamcarvalho/mpd-public#6]
Hi, I have a question regarding the dataloader here: https://github.com/jacarvalho/mpd-public/blob/main/mpd/trainer/train_loaders.py#L91. I am curious whether the argument `shuffle=True` is missing for the `train_dataloader`.

**Comment by joaoamcarvalho** [comment]
Hey, thanks for catching that. It should definitely have `shuffle=True`.

**Comment by OAHC2022** [comment]
thank you!!
